partiql-test-data/fail/semantic/primitives/time-constructor.ion

```diff
@@ -0,0 +1,55 @@
+semantic::{
```

Comment:
Similarly, all these `TIME` tests were labeled as parse errors by the Kotlin implementation. I felt they were better labeled as semantic errors.